### DISCUSSION TITLE
forgot-password

### DISCUSSION THREAD

**Discussion by tarunprakashsingh** [koush/scrypted#804] [General]
is there a way to reset the password? i had saved the login/password with browser and some how its deleted. cant find a way to reset the login password.

please help

**Comment by sebastiangoldstein** [comment]
Did you figure this out - having same issue.

**Reply by bjia56** [reply]
https://github.com/koush/scrypted/wiki/Lost-Password-and-User-Reset

**Comment by tarunprakashsingh** [comment]
nope....  live stream works but recording and notifications still dont work.

On Fri, Jul 7, 2023 at 10:35 AM sebastiangoldstein ***@***.***>
wrote:

> Did you figure this out - having same issue.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/koush/scrypted/discussions/804#discussioncomment-6385359>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFTGSNVXIN2L6GOQPYB2GDXPANBNANCNFSM6AAAAAAYFREMPI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Thanks
Tarun Singh

Like Us on Facebook CricketDepotOnline
<http://www.facebook.com/CricketDepotOnline>
Like Us on Facebook XTreme Cricket USA
<http://www.facebook.com/XtremeCricketUSA>

**Comment by tarunprakashsingh** [comment]
sorry ignore previous reply..
.https://github.com/koush/scrypted/wiki/Lost-Password-and-User-Reset

this will solve it

On Fri, Jul 7, 2023 at 10:39 AM Tarun Singh ***@***.***>
wrote:

> nope....  live stream works but recording and notifications still dont
> work.
>
> On Fri, Jul 7, 2023 at 10:35 AM sebastiangoldstein <
> ***@***.***> wrote:
>
>> Did you figure this out - having same issue.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/koush/scrypted/discussions/804#discussioncomment-6385359>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AAFTGSNVXIN2L6GOQPYB2GDXPANBNANCNFSM6AAAAAAYFREMPI>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>
>
> --
> Thanks
> Tarun Singh
>
> Like Us on Facebook CricketDepotOnline
> <http://www.facebook.com/CricketDepotOnline>
> Like Us on Facebook XTreme Cricket USA
> <http://www.facebook.com/XtremeCricketUSA>
>


-- 
Thanks
Tarun Singh

Like Us on Facebook CricketDepotOnline
<http://www.facebook.com/CricketDepotOnline>
Like Us on Facebook XTreme Cricket USA
<http://www.facebook.com/XtremeCricketUSA>
